molgenis-settings-ui/src/main/frontend/src/components/Settings.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div>
+    <h1>{{ 'plugin-title' | i18n }}</h1>
+    <form-component id="settings-form" :schema="schema"></form-component>
+  </div>
+</template>
+
+<script>
+  import { FormComponent } from '@molgenis/molgenis-ui-form'
+  import { GET_SETTINGS } from '../store/actions'
+  import { mapState } from 'vuex'
+
+  export default {
+    name: 'Settings',
+    components: {
+      FormComponent
+    },
+    created: function () {
+      this.$store.dispatch(GET_SETTINGS)
+    },
+    computed: {
+      ...mapState(['formFields']),
+      schema () {
+        return {
+          fields: this.formFields
```

Comment:
this can be written as `fields: this.$store.state.formFields`

Reply:
But but but... mapState is brilliant!

Reply:
I would agree with Mark that in these cases, 'this.$store.state.formFields' is more clear, less magic, less framework

Reply:
I'd go for 
```
  computed: mapState (state => { 
    schema: { 
      fields: state.formFields 
    } 
  })
```